ergo-chain-sync/src/client/types.rs

```diff
@@ -1,6 +1,8 @@
 use std::fmt::{Display, Formatter};
 
-#[derive(Debug, Clone)]
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Url(String);
```

Comment:
Should be enough for now, but I think we should add some kind of validation for such wrapped types, otherwise it will be not clear for user what went wrong is he set invalid URL or fs path.

Reply:
@kettlebell is there a way to check format of a string while parsing it?

Reply:
@oskin1 Yes, `isahc` internally uses the `isahc::http::Uri` type. I think if we wrap `Url(isahc::http::Uri)` instead we'll get built-in validation.

Reply:
Let's do this?

Reply:
Yes, I'll make the changes.